packages/cli/src/base-l2.test.ts

```diff
@@ -0,0 +1,117 @@
+import { Connection } from '@celo/connect'
+import { testWithAnvilL2 } from '@celo/dev-utils/lib/anvil-test'
+import { Config, ux } from '@oclif/core'
+import Web3 from 'web3'
+import { BaseCommand } from './base'
+import CustomHelp from './help'
+import { stripAnsiCodesFromNestedArray, testLocallyWithWeb3Node } from './test-utils/cliUtils'
+
+process.env.NO_SYNCCHECK = 'true'
+
+// doesnt use anvil because we are testing the connection to chain
+// doesnt use testLocally because that messes with the node connection
+describe('flags', () => {
+  let config: Config
+  class BasicCommand extends BaseCommand {
+    async run() {
+      console.log('Hello, world!')
+    }
+  }
+  beforeEach(async () => {
+    // copied from Commaand.run to load config
+    config = await Config.load(require.main?.filename || __dirname)
+  })
+  describe('--node  alfajores', () => {
+    it('it connects to 44787', async () => {
+      const command = new BasicCommand(['--node', 'alfajores'], config)
+      const runnerWeb3 = await command.getWeb3()
+      const connectdChain = await runnerWeb3.eth.getChainId()
+      expect(connectdChain).toBe(44787)
+    })
+  })
+  describe('--node  celo', () => {
+    it('it connects to 42220', async () => {
```

Comment:
testing the chain id not the url so we can swap it out later (for example for community rpc cluster?)